client/landing/jetpack-cloud/sections/backups/main.jsx

```diff
@@ -18,29 +18,31 @@ import {
 	getEventsInDailyBackup,
 	getMetaDiffForDailyBackup,
 } from './utils';
+import { applySiteOffset } from 'lib/site/timezone';
```

Comment:
sorted these imports during conflict resolution to make sure nothing was duplicated